Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1550,6 +1550,18 @@
     <!-- XTXT: Statistics FAQ text -->
     <string name="statistics_faq_text">"Weitere Informationen finden Sie in den FAQ:"</string>
 
+    <!-- Seven Day Hospitalization Statistics Card -->
+    <!-- XHED: Title for seven day hospitalization stats -->
+    <string name="statistics_seven_day_hospitalization_card_title">"7-Tage-Inzidenz Hospitalisierung"</string>
+    <!-- XTXT: Hospitalizations per 100.000 inhabitants -->
+    <string name="statistics_seven_day_hospitalization_per_inhabitants_text">"COVID-19-Hospitalisierungen je 100.000 Einwohner"</string>
+
```

Comment:
the string `statistics_seven_day_hospitalization_nationwide_text` is missing in the German translation file

Reply:
Oh, ok, I must have missed it :D